Maquetado/scss/styles.scss

```diff
@@ -0,0 +1,173 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: 24px;
+  justify-content: flex-end;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.techno-title {
+  font-size: 12px;
+}
+
+.type-tool {
+  font-size: 24px;
+  position: relative;
+  transform: translateX(100%);
+
+  &::before {
+    background-image: url('assets/triangle.svg');
+    background-repeat: no-repeat;
+    content: '';
+    height: 90px;
+    position: absolute;
+    transform: translateX(-50%);
+    top: -35px;
+    width: 100px;
+    z-index: -1;
+  }
+}
+
+.tools-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container-techno {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 100px;
+  padding: 50px;
+}
+
+.techno {
+  align-items: center;
+  background-color: $color-green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 100px 20px 10px 20pX;
+  position: relative;
+  width: 200px;
+
+  &::before {
+    border: 1px solid $color-green;
+    border-radius: 5px;
+    bottom: 10px;
+    content: '';
+    left: -10px;
+    right: 10px;
+    position: absolute;
+    top: -10px;
+    z-index: -1;
+  }
+}
+
+.logo-techn {
+  height: 100px;
+  margin: 10px;
+  width: 100px;
+}
+
+span {
+  font-size: 12px;
+  margin: 15px;
+  text-align: center;
+}
+
+.description {
+ display: flex;
+ font-size: 16px;
+ font-weight:bold;
+ margin: 5px;
+}
+
+.title {
+  display: flex;
+  font-size: 16px;
+  margin: 5px;
+}
+
+.nav-bar {
+  align-items: center;
+  box-shadow: 0px 11px 5px -5px $color-grey;
+  display: flex;
+  font-size: 24px;
+  flex-direction: row;
+  height:90px;
+  justify-content: space-between;
+  margin-left: 20px;
+}
+
+.send-button {
+	background-color: transparent;
+	border-radius: 8px;
+  border: 1px solid $color-white;
+	display: inline-block;
+	color: white;
+  cursor: pointer;
+	padding:15px 31px;
+	text-decoration:none;
+
+  &:hover {
+	  background-color:transparent;
+  }
+
+  &:active {
+	position:relative;
+  }
+}
+
+.email-input {
+  border:2px solid white;
+	border-radius:10px;
+	height: 28px;
+  margin-bottom: 20px;
+  margin-top: 20px;
```

Comment:
Merge both margins into `margin: 20px auto;` that will take both top and bottom like 20px and leaves the sides in it's default